created bug-bounty.md

yeqbfgxjiq opened the pull request:
Added a Bug Bounty page (`docs/contribute/bug-bounty.md`)
Linked it to the sidebar (`sidebar.js`)
> note: the repo is configured for local development to view the changes

yeqbfgxjiq committed:
created bug-bounty.md

lkngtn commented on website/siteConfig.js:
```diff
@@ -25,17 +25,17 @@ const siteConfig = {
 	// UNCOMMENT FOR PUBLISHING TO THE WEBSITE
-	url: 'http://1hive.org', // your website url
-	baseUrl: '/', // base url for your project
+	//url: 'http://1hive.org', // your website url
```
I don't think this is necessary for deployments anymore?

yeqbfgxjiq replied:
If it ain't broke, don't fix it lol

lkngtn committed:
Changing the url for publishing is not necessary

You don't need to make any changes to view locally anymore `yarn start` should just work, and not changing it each time for deployments seems simpler/safer.
